javascript/javascript1/week1/Exercise.js

```diff
@@ -0,0 +1,51 @@
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1988;
+const yearFuture = 2038;
+const age = yearFuture - yearOfBirth ;
+console.log("You will be " + age +" years old in " + yearFuture );
+
+
+
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth =  2019;
+const dogYearFuture =2045;
+const dogYear = dogYearFuture - dogYearOfBirth ;
+const shouldShowResultInDogYears = false;
+if (shouldShowResultInDogYears) {
+    console.log("Your dog will be "+  dogYear  + " dog years old in "+ dogYearFuture)
+} else {
+    console.log("Your dog will be "+ dogYear * 7 + " human years old in " + dogYearFuture) 
+};
+
+
+
+
+//Housey pricey (A house price estimator)
+
+const height = 10;
+const wide = 8;
+const deep = 10;
+volumeInMeters = deep * wide * height;
+gardenSizeInM2=100;
+console.log("Peter house Price Is "+(housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300));
```

Comment:
You are assigning a value to `housePrice` within the console.log statement. Although this approach works, it is better practice to separate the calculation and assignment for improved clarity.